src/generators/indexGenerator.ts

```diff
@@ -13,9 +13,12 @@ export function generateIndexFile(
     }
   );
 
-  const hasOperationsFile = clientDetails.operationGroups.some(
-    og => !og.isTopLevel
-  );
```

Comment:
why is this var gone? is it not used anywhere?

Reply:
Yeah it was not used at all, just cleaned it up